.github/workflows/platform-install-test.yml

```diff
@@ -0,0 +1,48 @@
+name: Platform Testing Installation
+on:
```

Comment:
@kdaud This looks good.However one question, is there a big difference with one of platform advanced installation testing ?

Reply:
The difference comes when selecting the type of installation during the set up, their respective feature files shows it clearly.

For Platform Advanced installation

```js
  
Feature: New Advanced Installation

  @selenium
  @initialSetup
  @advancedInstall
  Scenario: Advanced Platform Installation
    Given User selects Language on step one of Advanced Installation
    And User selects Advanced Installation Type
    When User enters all Advanced Installation steps
    Then System should run Advanced installation
```
For Platform Testing Installation

```js
Feature: New Testing Installation

  @selenium
  @initialSetup
  @testingInstall
  Scenario: Testing Platform Installation
    Given User selects Language on step one of Testing Installation
    And User selects Testing Installation Type
    When User enters all Testing Installation steps
    Then System should run Testing installation
```